### PR TITLE
created deleteTodo function

### PR DESCRIPTION
**Pull request by RQ16**

### CODE REVIEW

File: todo.js [hunk 1]
```diff
@@ -30,7 +30,17 @@ var todo = (function() {
       // should leave the input argument todos unchanged
       // return a new array, this should not contain any todo with an id of idToDelete
       // hint: array.filter
-      return state; //change me!
```
Comment (MohamedOmarii) [inline todo.js]: return what state ?!

**Comment by MohamedOmarii** [comment]
maybe we should delete the unused code which is commented

**Comment by RQ16** [comment]
I removed the code. Please merge

**Comment by MohamedOmarii** [comment]
goodjob guysh

**Pull request merged**